libs/backend/src/election_package/election_package_io.ts

```diff
@@ -146,10 +316,9 @@ export async function readElectionPackageFromUsb(
     return err('election_package_authentication_error');
   }
 
-  const electionPackage = await readElectionPackageFromBuffer(
-    await fs.readFile(filepathResult.ok())
-  );
-
+  const electionPackage = (
+    await readElectionPackageFromFile(filepathResult.ok())
+  ).unsafeUnwrap();
```

Comment:
If the containing function returns a `Result`, why are we unwrapping here? Can we just wrap the error?

Reply:
This is just preserving the existing behavior. I think the reasoning is that once an election package is signed by VxAdmin, it shouldn't have parsing errors when imported into another machine.